### PR TITLE
test cases for issues#6753

### LINKED ISSUE #6733: Add more test cases for SingletonServiceLoader's exception occur
- [x] Add test case when service is null by getServiceLoader
- [x] Add test case when service is not an interface by getServiceLoader
- [x] Add test case when service does not find by newServiceInstances

### PR DESCRIPTION
**Pull request by harvies**
Fixes #6733.

### CODE REVIEW

File: shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/spi/singleton/SingletonServiceLoaderTest.java [hunk 1]
```diff
@@ -26,9 +27,10 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
 
 public final class SingletonServiceLoaderTest {
-    
+
```
Comment (terrymanu) [inline shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/spi/singleton/SingletonServiceLoaderTest.java:33]: Please do not change the original intent, please update your IDEA code format settings

Comment (harvies) [reply shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/spi/singleton/SingletonServiceLoaderTest.java]: The idea configuration imported through the official website seems to be wrong.

**Comment by harvies** [comment]
https://github.com/apache/shardingsphere/issues/6741
Help me look at this issues.
